src/test/java/com/microsoft/sqlserver/jdbc/databasemetadata/DatabaseMetaDataTest.java

```diff
@@ -546,4 +546,21 @@ public void testPreparedStatementMetadataCaching() throws SQLException {
             }
         }
     }
+
+    @Test
+    public void testGetMaxConnections() throws SQLException {
+        try (Statement stmt = connection.createStatement(); ResultSet rs = stmt
+                .executeQuery("select maximum from sys.configurations where name = 'user connections'")) {
+            assert (null != rs);
+            rs.next();
+
+            DatabaseMetaData databaseMetaData = connection.getMetaData();
+            int maxConn = databaseMetaData.getMaxConnections();
+            
+            assert (0 != maxConn);
```

Comment:
why do we not allow this to be 0?

Reply:
It is impossible to connect to database and get response as 0 - at least 1 connection will be allowed when making this query on an active connection.

Reply:
Fixed